.dockerignore

```diff
@@ -14,7 +14,4 @@ mesg-dev
 bin/cli
 docs/
 scripts/
-docker-images/
-
-# prevent go build cache to break inside Docker on changes to systemservices.
-systemservices
```

Comment:
I was thinking to still keep this directory to keep sources of the services. Just to store everything as monorepo but it's not needed anymore for the core to run so I think this can stays

Reply:
Yeah let's keep it!

Reply:
@ilgooz could you revert this modif?

Reply:
done